GP_TOPIC: "GPS"
USR_TOPIC: "USR"

USR_MIN: 0
USR_MAX: 1
USR_THRESHOLD: 0.6
USR_PING_DURATION: 1.0

GP_COUNT: 5
GP_PREFIX: "GP-"